roles/openshift_facts/defaults/main.yml

```diff
@@ -14,6 +14,9 @@ openshift_cli_image: "{{ osm_image | default(openshift_cli_image_dict[openshift_
 osm_default_subdomain: "router.default.svc.cluster.local"
 openshift_master_default_subdomain: "{{ osm_default_subdomain }}"
 
+openshift_cluster_network_cidr: "{{ osm_cluster_network_cidr | default('10.128.0.0/14') }}"
```

Comment:
This will potentially break older clusters that need to scale up a new master.  We are slurping for the existing value because we changed the default at some point and changing the cidr isn't supported.

Reply:
@sdodson Based on our conversation last week about no longer scraping the master-config.yaml, do you see this as an issue?

Reply:
@michaelgugino @mtnbikenc I'm not opposed to putting the functionality back in if we can do it cleanly. It looks like we changed the default in 3.4 which is new enough that we should probably consider fixing this. If it were 3.1 or earlier I'd say lets skip it. Sorry, I thought we changed the default much earlier in the product lifecycle than this.